packages/design-system/stories/0-Text.stories.jsx

```diff
@@ -0,0 +1,49 @@
+import { Stack, Text } from '@chakra-ui/core';
+import React from 'react';
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
Should add a ESLint override for all stories so you dont need to do this in every file. 

Can make a .eslintrc file in the design-system/stories folder with this: 
```json
{
    "rules": {
      "import/no-default-export": "off",
      "@typescript-eslint/no-explicit-any": "off"
    }
}
```